numpy/lib/npyio.py

```diff
@@ -1167,9 +1165,13 @@ def loadtxt(fname, dtype=float, comments='#', delimiter=None,
 
     Notes
     -----
-    This function aims to be a fast reader for simply formatted files.  The
-    `genfromtxt` function provides more sophisticated handling of, e.g.,
-    lines with missing values.
+    * This function aims to be a fast reader for simply formatted files.  The
+      `genfromtxt` function provides more sophisticated handling of, e.g.,
+      lines with missing values.
+    * Each row in the input text file must have the same number of values to be
+      able to read all values. If all rows do not have same number of values, a
+      subset of up to n-columns (where n is the least number of values present
+      in all rows) can be read by specifying the columns via `usecols`.
```

Comment:
Nitpick: these two sections don't need to be bulleted.  Regular paragraphs with a blank line between them should be fine.  Also, change `n-columns` to `n columns`.
```suggestion
    This function aims to be a fast reader for simply formatted files.  The
    `genfromtxt` function provides more sophisticated handling of, e.g.,
    lines with missing values.

    Each row in the input text file must have the same number of values to be
    able to read all values. If all rows do not have same number of values, a
    subset of up to n columns (where n is the least number of values present
    in all rows) can be read by specifying the columns via `usecols`.
```